docs/_docs/modifiers/modifiers.md

```diff
@@ -256,56 +256,86 @@ VariantTestDoc.Square(
 
 ### Component replacement {#comp-replace}
 
-With a [`Composable`][2] of your choice, an element of a design can be replaced
-while retaining the style information provided by the designer. The difference
-between replacing versus adding descendants (as shown in [Creating lists][5] is
-that a replaced Figma design element isn't rendered.
+An node in a design can be replaced with a [`Composable`][2] of your choice while
+retaining the style information provided by the designer. The difference between replacing
```

Comment:
It's an odd sentence, I feel like it would confuse more than help.

Reply:
It explains the difference between replacing a node and replacing the children of a node. It's not the best sentence I agree but I don't want to hold up the release for this. Feel free to improve on it after the RC is made.